code/dynamic_programming/src/longest_common_substring/Longest_Common_Substring.cpp

```diff
@@ -0,0 +1,71 @@
+// Space Complexity: O(n)
+// Time Complexity: O(m*n) 
+
+#include <string>
+#include <iostream>
+#include <vector> 
+
+std::string LongestCommonSubstring(std::string str1, std::string str2) 
```

Comment:
Could you fix the code according to our [C++ Style Guide](https://github.com/OpenGenus/cosmos/tree/master/guides/coding_style/c%2B%2B)?